pkg/commands/workload_get_test.go

```diff
@@ -359,6 +367,99 @@ Source
 type:    image
 image:   docker.io/library/nginx:latest
 
+Supply Chain resources not found
+
+No pods found for workload.
+`,
+		},
+		{
+			Name: "show resources",
+			Args: []string{workloadName},
+			GivenObjects: []client.Object{
+				parent.
+					SpecDie(func(d *diecartov1alpha1.WorkloadSpecDie) {
+						d.Source(&cartov1alpha1.Source{
+							Git: &cartov1alpha1.GitSource{
+								URL: url,
+								Ref: cartov1alpha1.GitRef{
+									Branch: "master",
+									Tag:    "v1.0.0",
+									Commit: "abcdef",
+								},
+							},
+						})
+					}).
+					StatusDie(func(d *diecartov1alpha1.WorkloadStatusDie) {
+						d.ConditionsDie(
+							diecartov1alpha1.WorkloadConditionReadyBlank.
+								Status(metav1.ConditionFalse).Reason("OopsieDoodle").
+								Message("a hopefully informative message about what went wrong").
+								LastTransitionTime(metav1.Time{
+									Time: time.Date(2019, 6, 29, 01, 44, 05, 0, time.UTC),
+								}),
+						)
+						d.Resources([]cartov1alpha1.RealizedResource{
+							{
+								Name: "source-provider",
+								Conditions: []metav1.Condition{
```

Comment:
You can create ConditionDie similar to status conditions on realized resource conditions. Might make this testing block lot smaller.